lib/index.js

```diff
@@ -17476,7 +17476,7 @@ async function run() {
         } else if (distro === "arch") {
           // partial upgrades are not supported so also upgrade everything
           await execShellCommand(optionalSudoPrefix + 'pacman -Syu --noconfirm xz openssh');
-        } else if (distro === "fedora") {
+        } else if (distro === "fedora" || distro === "almalinux") {
```

Comment:
The only actual change

Reply:
@SanderVocke You will want to make the change in [`src/index.js`](https://github.com/SanderVocke/action-tmate/blob/9a9c1bee5a8f3bcbd4540a90e0630b550e7f57ab/src/index.js#L93) instead, and re-generate `lib/index.js` via `npm run build` (see e.g. https://github.com/SanderVocke/action-tmate/commit/192e28923a93b013116c680146f3c9b732b23a94).